webapp/src/css/inbox.less

```diff
@@ -1421,8 +1421,8 @@ mm-search-bar {
         font-size: @font-extra-extra-large;
       }
 
-      .fast-action-resource-icon {
-        line-height: 60px;
+      .resource-icon {
+        display: block;
```

Comment:
This is how it displays for me with this fix - the icons seem top aligned
<img width="470" alt="Screenshot 2024-08-27 at 12 16 45 PM" src="https://github.com/user-attachments/assets/f309df8c-6799-4cd8-93f0-984b6faf01b7">

This is how it was originally intended - the icons seem aligned middle
<img width="437" alt="Screenshot 2024-08-27 at 12 15 38 PM" src="https://github.com/user-attachments/assets/b2db9ae7-98b0-4afb-8b1b-8f611964a176">

The icons are uploaded into the config, since we don't have control over what is uploaded for icons. Better align the container

<img width="1043" alt="Screenshot 2024-08-27 at 12 20 09 PM" src="https://github.com/user-attachments/assets/e393759b-16e0-4337-bae0-d697032a0f0d">

Reply:
In most places in the app, icons are centered aligned to the text (1 line or 2 lines of text)
<img width="765" alt="Screenshot 2024-08-27 at 12 23 22 PM" src="https://github.com/user-attachments/assets/0d35da76-37bd-4cbf-8ce3-72c38ddedb5a">